modules/glance/hm.nix

```
@@ -30,9 +31,14 @@
                     0;
                 l = (max + min) / 2;
                 s =
-                  if delta == 0 then 0 else delta / (100 - lib.max (2 * l - 100) (100 - (2 * l)));
+                  if delta == 0 then
+                    0
+                  else
+                    100 * delta / (100 - lib.max (2 * l - 100) (100 - (2 * l)));
               in
-              "${builtins.toString h} ${builtins.toString s} ${builtins.toString l}";
+              "${builtins.toString (builtins.floor (h + 0.5))} ${
+                builtins.toString (builtins.floor (s + 0.5))
+              } ${builtins.toString (builtins.floor (l + 0.5))}";
```

`builtins.toString` is already in the prelude. Also, consider reformatting:

```suggestion
              "${
                toString (builtins.floor (h + 0.5))
              } ${
                toString (builtins.floor (s + 0.5))
              } ${
                toString (builtins.floor (l + 0.5))
              }";
```

---

Alternative:

```suggestion
              let
                roundToString = value: builtins.toString (builtins.floor (value + 0.5));
              in
                lib.concatMapStringsSep "  " roundToString [ h s l ]
```